examples/zurb5/app.py

```diff
@@ -0,0 +1,151 @@
+import os
+import os.path as op
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+
+import flask_admin as admin
+from flask_admin.contrib.sqla import ModelView
+
+
+# Create application
+app = Flask(__name__)
+
+# Create dummy secrey key so we can use sessions
+app.config['SECRET_KEY'] = '123456790'
+
+# Create in-memory database
+app.config['DATABASE_FILE'] = 'sample_db.sqlite'
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///' + app.config['DATABASE_FILE']
+app.config['SQLALCHEMY_ECHO'] = True
+db = SQLAlchemy(app)
+
+
+
+# Models
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.Unicode(64))
+    email = db.Column(db.Unicode(64))
+
+    def __unicode__(self):
+        return self.name
+
+
+class Page(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.Unicode(64))
+    content = db.Column(db.UnicodeText)
+
+    def __unicode__(self):
+        return self.name
+
+
+# Customized admin interface
+class CustomView(ModelView):
+    list_template = 'list.html'
```

Comment:
I checked it out from your fork and used ```python setup.py install```.

This code assumes there following files exist:
* examples/zurb5/templates/create.html
* examples/zurb5/templates/edit.html
* examples/zurb5/templates/list.html

To get this example to work, you can just remove ```class CustomView(ModelView):```.